source-analysis.md

```diff
@@ -1,36 +1,47 @@
+
 * Who, or what organization, is responsible for hosting the source (if online), who gathered the information?
+This was a personal project of two "obelisks fans". There names are Hiroyuki Nagase and Shoji Okamoto. It seems that Hiroyuki Nagase did most of the work visiting each obelisk and collecting the data about them.
 
 ---
 
 ## Authorship
 
 * Is there any information available discussing the process by which the information was gathered, or how the digital project was constructed?
-* How transparent is the project about the choices they made in gathering and transforming the data for digitalization? [Click here for an example of a an excellent 'About' page from a database about Anglo-Saxons](http://www.pase.ac.uk/about/index.html)
+The author of the website states that he went to these obelisks to observe and take data on there present condition. It seems that some of the information has been taken from Wikipedia, Google Maps, and linked to different websites related to each individual obelisk.
+
+* How transparent is the project about the choices they made in gathering and transforming the data for digitalization?
+The project seems to be fairly transparent. The author has sections broken down into different discusses about why each obelisk was chosen and how they were studied.  The author describes the short falls of the website right away, saying that it is not complete nor 100% accurate.
+
 * Do the organizations or individuals involved in gathering and publishing your data set have any stated goals or agendas?
+There goal was to collect the most information they could on their travels about the obelisk around the world. They called themselves "obelisks fans".
+
 * Do the organizations or individuals involved seem to have any unstated agendas or goals?
+Just seems to be a bunch of Japanese obelisk enthusiasts.
+
 * If a 3rd party funded the project, who are they? Do they have any particular perspective?
+N/A
 
 ---
 
 ## Data Analysis
 
 * Do the datasets involved reflect the viewpoints of particular groups or institutions, and if so, who?
+The dataset reflects the work done by Hiroyuki Nagase and Shoji Okamoto through their travels looking for these obelisks.
+
 * What kinds of categorizations do you find in the data, and do they seem reasonable?
+The website starts with a list of obelisks which the authors have visited. Then are linked to a page with details about the individual obelisks. Included is the dates of the obelisks' origin, measurements, weight, stone type, location and background. Some of the pages have links to museums that are associated with each individual obelisk.   
+
 * Do the means of data collection or categorization affect how the data is constructed in any way?
+Yes, the data has been collected by a group of people who seem to not be in field of social sciences but just enthusiasts. The categorization has been affected by the personal preferences of the authors.
```

Comment:
Might this also mean that they do not record certain kinds of information in a systematic (and searchable) manner, for example, the original texts on any obelisks? In fact, they seem to have little systematic data at all, aside from liking to their locations and having a text description.